src/content/docs/reference/recipe.mdx

```diff
@@ -3,7 +3,23 @@ title: recipe.yml
 description: A reference page in my new Starlight docs site.
 ---
 
-https://diataxis.fr/reference/
+## name
 
-- technical description of recipe
-- list of all keys
\ No newline at end of file
+The image name. Used when publishing to GHCR as `ghcr.io/<user>/<name>`
+
+## description
+
+The image description. Published to GHCR in the image metadata
+
+## base-image
+
+The image upon which all your customizations will be based. It can be any of the images listed on the [uBlue website](https://universal-blue.org/images/) or any BlueBuild-produced image.
```

Comment:
Should mention OCI standard.

Should be more open to being any OCI image, with a note that 'Only atomic Fedora images and those based on them are officially supported. Universal Blue is recommended, [see list].'